themes/default/content/docs/concepts/vs/terraform/_index.md

```diff
@@ -33,7 +33,7 @@ Pulumi’s universal infrastructure as code platform helps teams tame the cloud
 
 ## What is Terraform?
 
-Terraform provides open-source [infrastructure as code](/what-is/what-is-infrastructure-as-code/) software for cloud service management with a consistent CLI workflow. Terraform allows you to write, plan, and apply changes to deliver infrastructure as code.
+Terraform provides [infrastructure as code](/what-is/what-is-infrastructure-as-code/) software for cloud service management with a consistent CLI workflow. Terraform allows you to write, plan, and apply changes to deliver infrastructure as code.
```

Comment:
Perhaps something like:

"Terraform is a free, source available infrastructure as code tool from HashiCorp" ...  bla bla bla ...